### ISSUE TITLE
Jet Engine - add data store info to form submission *IMPORTANT

### ISSUE THREAD

**Issue by perks98** [CrocoBlock/suggestions#2463]
Can you look into making it so that any 'data store' can be processed and used in form submissions. For instance, you use the 'data store' feature as a favourites for custom post type then on the favourites page be able to submit a form which tells the business the list of items either by name, id or link in an email of what they liked, so it can be used as a request an estimate or quote.

I have a bridal shop customer looking to add a feature for a customer to add gowns to their wishlist then have an appointment form, so it tells the business what items they want to try and the rest of the form data like date and time.

Hope that makes sense.

**Comment by taidang26** [comment]
Its applicability is really high. I have a website about recruitment and also want to apply this feature to candidates who have submitted resumes to employers and manage submitted resumes.
Please consider adding this functionality.

**Comment by naturalweb-business** [comment]
Absolutely necessary +1

**Comment by DouglasFletcher** [comment]
Necessary +1

**Comment by osamu-wakabayashi** [comment]
> Its applicability is really high. I have a website about recruitment and also want to apply this feature to candidates who have submitted resumes to employers and manage submitted resumes. Please consider adding this functionality.

@taidang26, I am working on a volunteer management site with JetEngine.  It sounds similar to your project that companies post job opportunities and job candidates apply.  Can we connect and share our experiences?

**Comment by taidang26** [comment]
> > Its applicability is really high. I have a website about recruitment and also want to apply this feature to candidates who have submitted resumes to employers and manage submitted resumes. Please consider adding this functionality.
> 
> @taidang26, I am working on a volunteer management site with JetEngine. It sounds similar to your project that companies post job opportunities and job candidates apply. Can we connect and share our experiences?

Yes, of course. Do you need any help? I have completed my website and am happy to share it with you.

**Comment by osamu-wakabayashi** [comment]
It will be nice if we can get connected.  Can you send me your email? You can reach me at osamu@zenideas.ca

**Comment by bulutemre** [comment]
+1 yes! we also need that. Even if just titles from this favourite list items could be sent with a form, it would be amazing.

**Comment by giacozama** [comment]
+1

**Comment by tamir1947** [comment]
> > > Its applicability is really high. I have a website about recruitment and also want to apply this feature to candidates who have submitted resumes to employers and manage submitted resumes. Please consider adding this functionality.
> > 
> > 
> > @taidang26, I am working on a volunteer management site with JetEngine. It sounds similar to your project that companies post job opportunities and job candidates apply. Can we connect and share our experiences?
> 
> Yes, of course. Do you need any help? I have completed my website and am happy to share it with you.

I would love your help, how did you solve it
Thanks

**Comment by imihaiart** [comment]
+1 

Any updates on this? Someone found any solution? Also would be needed a quantity input for every data store item, just like the Woocommerce one. So the users can ask quotes for different quantity for different items in their data store.

**Comment by cazper57** [comment]
+1

**Comment by andresmuttio** [comment]
+1

**Comment by EnvioMaik** [comment]
+999 this would open up so many opportunities, any progress? Doesn't seems hard to implement

**Comment by geeko1936** [comment]
+1

**Comment by aalexandr21** [comment]
was added and released in JetEngine 3.7.2
